app/src/main/java/com/asfoundation/wallet/service/currencies/CurrenciesDatabase.kt

```diff
@@ -0,0 +1,28 @@
+package com.asfoundation.wallet.service.currencies
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import com.asfoundation.wallet.change_currency.FiatCurrenciesDao
+import com.asfoundation.wallet.change_currency.FiatCurrency
+
+@Database(entities = [CurrencyConversionRateEntity::class, FiatCurrency::class], version = 2)
+abstract class CurrenciesDatabase : RoomDatabase() {
+
+  abstract fun currencyConversionRatesDao(): CurrencyConversionRatesDao
+
+  abstract fun fiatCurrenciesDao(): FiatCurrenciesDao
+
+  companion object {
+    val MIGRATION_1_2: Migration = object : Migration(1, 2) {
+      override fun migrate(database: SupportSQLiteDatabase) {
+        database.execSQL(
+            "CREATE TABLE IF NOT EXISTS fiat_currencies (currency INTEGER PRIMARY KEY, flag TEXT , label TEXT , sign TEXT)")
+//        database.execSQL(
+//            "CREATE TABLE IF NOT EXISTS 'FiatCurrency' ('currency' TEXT NOT "
+//                + "NULL, 'flag' TEXT, 'label' TEXT, 'sign' TEXT, PRIMARY KEY(currency))")
+      }
+    }
+  }
```

Comment:
Is this migration really needed? I think if it is a new db, it does not need a migration, but I'm not 100% sure.